### DISCUSSION TITLE
Get first name and last name from Google auth

### DISCUSSION THREAD

**Discussion by jonas-dongowski** [pocketbase/pocketbase#932] [Q&A]
Hey there. For some reason when a new user is created through Google auth the name is not being updated.
Is there a way for me to get that data either on user creation inside PocketBase so it's part of the profile or at runtime through some API call to Google?

**Comment by ganigeorgiev** [comment]
The Google profile name should be available in the user auth response under the `meta` object that is returned to the client.
The OAuth2 profile name will not automatically be assigned to your auth collection record because it is dynamic and we can't be sure if there is a registered "name" field in your collection (it could be also named differently, eg. `fullName`), so it's up to the developers to update the PocketBase user model with the returned `meta.name` value if they want to.

If after a successfull Google OAuth2 response `meta.name` is empty, then either the API has changed or we have a bug in our response mapper - https://github.com/pocketbase/pocketbase/blob/master/tools/auth/google.go#L34.

**Reply by jonas-dongowski** [reply]
Awesome, thank you. I'm gonna try that.
